processing/data_collection/gazette/spiders/mg_governador_valadares.py

```diff
@@ -0,0 +1,120 @@
+import ast
+from datetime import datetime, date
+import json
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgGovernadorValadares(BaseGazetteSpider):
+    TERRITORY_ID = "3127701"
+    name = "mg_governador_valadares"
+    allowed_domains = ["valadares.mg.gov.br"]
+    start_urls = [
+        "https://www.valadares.mg.gov.br/diario-eletronico/caderno/diario-oficial-eletronico/1"
+    ]
+
+    ITEMS_PER_PAGE = "100"
+    BASE_URL = "https://www.valadares.mg.gov.br/"
+
+    path = ""
+    current_page = 0
+
+    def parse(self, response):
+        scripts = response.xpath("//script//@src").extract()
+        endpoint = next(path for path in scripts if "diel_diel_lis" in path)
+        self.path = endpoint
```

Comment:
This is not wrong. But I would say to move this to another function with a good name. Maybe, something like `get_script_path_to_request_items`. I'm not sure this a good name, but I think something like this will give us a even better code!